lib/spectator/version.rb

```diff
@@ -1,3 +1,3 @@
 module Spectator
-  VERSION = '0.1.0'.freeze
+  VERSION = '0.1.1'.freeze
```

Comment:
Is there anyway we can extract the version from the tag name like we do for java projects? That would avoid duplication and ensure we do not have a mismatch between the tag and the version published.

As a side note, does the gem repo reject protect against overwriting a previous version?

Reply:
I was looking into that. The approach I found was using `rake release` to create the tag based on the spec version (so it would do the `git tag -a` with the version extracted) and push the tag to github, then travis will take it from there.